evap/staff/views.py

```diff
@@ -350,7 +357,7 @@ def semester_lottery(request, semester_id):
             if not courses.exclude(voters=user).exists():
                 eligible.append(user)
 
-        winners = random.sample(eligible, min([form.cleaned_data['number_of_winners'], len(eligible)]))
+        winners = sample(eligible, min([form.cleaned_data['number_of_winners'], len(eligible)]))
```

Comment:
It was suggested by PyCharm. Also makes the line shorter. random.sample might be easier to understand though, I see your point.